pkg/controller/resourcequota/replenishment_controller.go

```diff
@@ -273,18 +196,3 @@ func IsUnhandledGroupKindError(err error) bool {
 	_, ok := err.(unhandledKindErr)
 	return ok
 }
-
-// UnionReplenishmentControllerFactory iterates through its constituent factories ignoring, UnhandledGroupKindErrors
-// returning the first success or failure it hits.  If there are no hits either way, it return an UnhandledGroupKind error
-type UnionReplenishmentControllerFactory []ReplenishmentControllerFactory
```

Comment:
Since I was deleting the concept of a replenishment controller factory, I got rid of this too. I can see if there's some type I can add back that will achieve a similar effect.